spec/final-frame-spec.js

```diff
@@ -0,0 +1,66 @@
+describe('Final Frame', function () {
+  var finalFrame
+
+  beforeEach(function () {
+    finalFrame = new FinalFrame()
+  })
+
+  describe('A final frame', function () {
+    it('should inherit from a regular frame', function () {
+      spyOn(Frame.prototype, 'getBowls')
+      finalFrame.getBowls()
+      expect(Frame.prototype.getBowls).toHaveBeenCalled()
```

Comment:
Admirable that you're so thorough with your testing but remember that we should test a program's behaviour, _not_ its structure. Testing inheritance is 100% testing structure and makes it harder to change the structure of your app later.

A good question to ask is: "what sorts of changes would break this test? Do I want them to?" – if yes, it's a good test.